misc/error_injector/celix_properties/celix_properties_ei.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+ */
+
+#include <celix_properties_ei.h>
```

Comment:
nitpick: `#include "celix_properties_ei.h"`

Reply:
Oops! I should have followed Celix's existing convention, and it will be corrected.

In John Lakos's Large Scale C++, it is suggested that angle brackets instead of double quotes should be used exclusively in include directives. I follow his methods in my day-time job. However, existing convention should always take priority.